x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/ArrayStateTests.java

```diff
@@ -171,44 +170,44 @@ private void setAll(AbstractArrayState state, List<Object> values, int offset) {
 
     private AbstractArrayState newState() {
         return switch (type) {
-            case INTEGER -> new IntArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
-            case LONG -> new LongArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
-            case FLOAT -> new FloatArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
-            case DOUBLE -> new DoubleArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
-            case BOOLEAN -> new BooleanArrayState(BigArrays.NON_RECYCLING_INSTANCE, false);
-            case IP -> new IpArrayState(BigArrays.NON_RECYCLING_INSTANCE, new BytesRef(new byte[16]));
+            case "integer" -> new IntArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
+            case "long" -> new LongArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
+            case "float" -> new FloatArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
+            case "double" -> new DoubleArrayState(BigArrays.NON_RECYCLING_INSTANCE, 1);
+            case "boolean" -> new BooleanArrayState(BigArrays.NON_RECYCLING_INSTANCE, false);
+            case "ip" -> new IpArrayState(BigArrays.NON_RECYCLING_INSTANCE, new BytesRef(new byte[16]));
             default -> throw new IllegalArgumentException();
         };
     }
 
     private void set(AbstractArrayState state, int groupId, Object value) {
         switch (type) {
-            case INTEGER -> ((IntArrayState) state).set(groupId, (Integer) value);
-            case LONG -> ((LongArrayState) state).set(groupId, (Long) value);
-            case FLOAT -> ((FloatArrayState) state).set(groupId, (Float) value);
-            case DOUBLE -> ((DoubleArrayState) state).set(groupId, (Double) value);
-            case BOOLEAN -> ((BooleanArrayState) state).set(groupId, (Boolean) value);
-            case IP -> ((IpArrayState) state).set(groupId, (BytesRef) value);
+            case "integer" -> ((IntArrayState) state).set(groupId, (Integer) value);
+            case "long" -> ((LongArrayState) state).set(groupId, (Long) value);
+            case "float" -> ((FloatArrayState) state).set(groupId, (Float) value);
```

Comment:
From what I see, this is only used by `MaxFloatAggregator` and `MinFloatAggregator`. Which then are not used by the `Max.java` nor `Min.java` function definitions.

This is a WIP to implement Floats in ESQL, right? Otherwise I'd check what to do with it